libraries/ESP_NOW/src/ESP32_NOW.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "esp_wifi_types.h"
+#include "Print.h"
+#include "esp_now.h"
+
+class ESP_NOW_Peer {
+private:
+  uint8_t mac[6];
+  uint8_t chan;
+  wifi_interface_t ifc;
+  bool encrypt;
+  uint8_t key[16];
+
+protected:
+  bool added;
+  bool add();
+  bool remove();
+  size_t send(const uint8_t * data, int len);
```

Comment:
Wouldn't it be better for these methods to be public ? This way we can call them directly from the class that inherits this and it won't require something like `bool add_peer() { return add(); }`

Reply:
you should not need to do that. peer should be added in the implementation's begin method.